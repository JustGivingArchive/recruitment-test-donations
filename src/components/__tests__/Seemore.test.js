import React from 'react';
import renderer from 'react-test-renderer';
import SeeMore from '../Seemore.js';

describe('Test Intro component', () => {

  it('matches snapshot', () => {
    const tree = renderer.create(
      <SeeMore toggle={ () => null }/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})
