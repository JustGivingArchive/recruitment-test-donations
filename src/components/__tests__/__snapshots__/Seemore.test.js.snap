exports[`Test Intro component matches snapshot 1`] = `
<div
  className="panel hoverable"
  onClick={[Function]}>
  <h2
    className="paneltextlarge">
    See other Charities
  </h2>
</div>
`;
